controllers/project_controller.js

```diff
@@ -12,24 +12,15 @@ const controller = {
     }
   },
   newProjectForm(req, res) {
-    res.render("./project-detail/project_new.ejs");
+    console.log(req.session);
+    res.render("./projects/project_new.ejs");
   },
   async create(req, res) {
-    // const personalData = req.body;
-    // await currentUser.updateOne({
-    //   job_title: personalData.job_title,
-    //   postal_code: personalData.postal_code,
-    //   city: personalData.city,
-    //   linkedIn: personalData.linkedIn,
-    //   skills: personalData.skills,
-    //   about_me: personalData.about_me,
-    //   profile_photos_url: personalData.profile_photos_url,
-    //   cover_photos_url: personalData.cover_photos_url,
-    // });
-    // res.redirect(`/profile/${currentUser._id}`);
     const projectData = req.body;
     const currentUser = await Users.findById(`${req.session?.currentUser._id}`);
-    res.send("Created Project");
+    projectData.author_id = currentUser._id;
+    await Projects.create(projectData);
+    res.redirect(`/profiles/${currentUser._id}`);
```

Comment:
Why not redirect to project page directly?

Reply:
I redirect to profile page because ytd I think the place to create project is on profile page. 
Hm... BUT ... yeah... should go to project page to see it okay or not . =)) I will update.